light-node/src/bin/light-node/main.rs

```diff
@@ -1,4 +1,4 @@
-//! Main entry point for LightNode
+//! Main entry point for light-node
```

Comment:
I'm not sure why this shows up as a separate crate:
![image](https://user-images.githubusercontent.com/7441766/87785362-551e6300-c838-11ea-966e-695b49f2ba89.png)